Configuration/python/processingUtilities.py

```diff
@@ -979,6 +990,34 @@ def customizeMINIAODElectronVID(process, collections, usedCollections):
 
     return process
 
+def customizeMINIAODPhotonVID(process, collections, usedCollections):
+    if not hasattr (collections, "photons"):
```

Comment:
I think this should be
`if not hasattr (collections, "photons") or "photons" not in usedCollections:`
and similar for `customizeMINIAODElectronVID`.

Everyone has photons/electrons defined in `collections`, but I'm pretty sure this check should just skip adding CPU-costly VID modules when you're not cutting on or plotting these things.

I'm ~pretty sure but it's worth a test.